contracts/Comptroller/Comptroller.sol

```diff
@@ -1017,6 +1021,31 @@ contract Comptroller is ComptrollerV9Storage, ComptrollerInterfaceG2, Comptrolle
         allMarkets.push(vToken);
     }
 
+    function _initializeMarket(address vToken) internal {
+        uint32 blockNumber = safe32(getBlockNumber(), "block number exceeds 32 bits");
+
+        VenusMarketState storage supplyState = venusSupplyState[vToken];
```

Comment:
I'm curious, making this storage creates a pointer back to `venusSupplyState[vToken]`?

Reply:
Yes @coreyar , and I have added a test case for the `_initializeMarket` and it's working fine. you can take look [here](https://github.com/VenusProtocol/venus-protocol/pull/140/files#diff-1b31e3c7483dd3f63bcec1045cbfed6e031fb6e7fe5c9e5a249a75c31304895dR34).